docs/components/stepper.md

```diff
@@ -0,0 +1,130 @@
+# Stepper
+
+**KStepper** - A basic Stepper component
+
+<KStepper :steps="defaultItems" />
+
+```html
+<KStepper :steps="steps" />
+```
+
+## Props
+
+### steps
+
+An array of `KSteps`. Each `KStep` has a `label` and an optional `state`.
+
+<div>
+  <KStepper :steps="[
+      { label: 'Step  a long long long long time ago', state: 'completed' },
+      { label: 'in a galaxy far far away', state: 'completed' },
+      { label: 'Kongponents were battling in space and', state: 'pending' },
+      { label: 'fighting robots and space monsters with lots of explosions' }
+    ]"
+  />
+</div>
+
+```html
+<KStepper :steps="[
+    { label: 'Step  a long long long long time ago', state: 'completed' },
+    { label: 'in a galaxy far far away', state: 'completed' },
+    { label: 'Kongponents were battling in space and', state: 'pending' },
+    { label: 'fighting robots and space monsters with lots of explosions' }
+  ]"
+/>
+```
+
+### maxLabelWidth
+
+The width of step labels (default is `170`). Currently we support numbers (will be converted to `px`), `auto`, and percentages for values.
```

Comment:
I know the default is actually a `number` of `170` but can we call out here in the docs that it's pixels just so there's no confusion?
```suggestion
The width of step labels (default is `170px`). We support numbers (will be converted to `px`), `auto`, and percentages (e.g. `25%`) for values.
```